### ISSUE TITLE
MathJax公式渲染问题

### ISSUE THREAD

**Issue by zhanghuimeng** [yscoder/hexo-theme-indigo#462]
行内和行间公式都无法渲染，源文件中已经在行间公式周围加上`{% raw %} ... {% endraw %}`标签了，网页似乎也加载了MathJax。

我只在post的front-matter中开启了`mathjax: true`，不过在主题的`_config.yml`中打开mathjax效果也相同。

[示例链接](https://zhanghuimeng.github.io/post/why-we-should-compute-sigmoid-and-softmax-with-cross-entropy/)

**Comment by zhanghuimeng** [comment]
我觉得可能是renderer的问题，把[hexo-renderer-markdown-it](https://github.com/hexojs/hexo-renderer-markdown-it)换回hexo-renderer-marked后，发现可以开启数字TOC了（原来不行），但是公式仍然无法渲染。

**Comment by plmsmile** [comment]
我使用pandoc进行渲染

**Comment by zhanghuimeng** [comment]
谢谢您的意见，我有空会尝试一下的。看起来pandoc比其他renderer要稍微麻烦一些。我最近已经换成了其他主题，并尝试把renderer换成了[hexo-renderer-markdown-it-plus](https://github.com/CHENXCHEN/hexo-renderer-markdown-it-plus)，使用katex渲染数学公式，但是公式仍然有一些错位……

**Comment by plmsmile** [comment]
我写了一个pandoc的简易使用教程：[pandoc配置数学公式](https://plmsmile.github.io/2017/10/16/24-hexo-problems/#%E9%85%8D%E7%BD%AE%E6%95%B0%E5%AD%A6%E5%85%AC%E5%BC%8F)。可以作为一个参考。

**Comment by MakerGYT** [comment]
转化为图片是最稳定的方法，试试[markdown-it-latex2img](https://github.com/MakerGYT/markdown-it-latex2img#Hexo)
